gpjax/likelihoods.py

```diff
@@ -55,8 +55,8 @@ def __init__(
         """Initializes the likelihood.
 
         Args:
-            num_datapoints: the number of data points.
-            integrator: The integrator to be used for computing expected log
+            num_datapoints (int): the number of data points.
```

Comment:
I'd personally avoid including the type in the docstrings: it's already in the arguments type annotations and it is then repeated twice in the documentation (mkdocstrings automatically fills the docstring with the type from the annotations). It's also consistent with flax style.

I haven't updated all docstrings yet but it's on the todo list. But no problem leaving it as it is right now!